### ISSUE TITLE
dnvm use alias doesn't work when alias points to coreclr

### ISSUE THREAD

**Issue by victorhurdugaci** [aspnet/dnvm#175]
Even though the alias points to `coreclr`, when I run `kvm use <alias>` it tries to use the `clr` runtime

```
D:\aspnet\dnx>dnvm upgrade -r coreclr -amd64
Downloading dnx-coreclr-win-x64.1.0.0-beta4-11237 from https://www.myget.org/F/aspnetvnext/api/v2
Installing to C:\Users\victorhu\.dnx\runtimes\dnx-coreclr-win-x64.1.0.0-beta4-11237
Adding C:\Users\victorhu\.dnx\runtimes\dnx-coreclr-win-x64.1.0.0-beta4-11237\bin to process PATH
............

D:\aspnet\dnx>dnvm list
Active Version           Runtime Architecture Location                        Alias
------ -------           ------- ------------ --------                        -----
       1.0.0-beta4-11237 clr     x86          C:\Users\victorhu\.dnx\runtimes
  *    1.0.0-beta4-11237 coreclr x64          C:\Users\victorhu\.dnx\runtimes default

D:\aspnet\dnx>dnvm alias test 1.0.0-beta4-11237 -r coreclr -arch x64
Setting alias 'test' to 'dnx-coreclr-win-x64.1.0.0-beta4-11237'

D:\aspnet\dnx>dnvm list

Active Version           Runtime Architecture Location                        Alias
------ -------           ------- ------------ --------                        -----
       1.0.0-beta4-11237 clr     x86          C:\Users\victorhu\.dnx\runtimes
  *    1.0.0-beta4-11237 coreclr x64          C:\Users\victorhu\.dnx\runtimes default, test

D:\aspnet\dnx>dnvm use test
Cannot find dnx-clr-win-x64.1.0.0-beta4-11237, do you need to run 'dnvm install test'?
At C:\Users\victorhu\.dnx\bin\dnvm.ps1:1069 char:9
+         throw "Cannot find $runtimeFullName, do you need to run '$CommandName in ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Cannot find dnx... install test'?:String) [], RuntimeException
    + FullyQualifiedErrorId : Cannot find dnx-clr-win-x64.1.0.0-beta4-11237, do you need to run 'dnvm install test'?

```

**Comment by anurse** [comment]
As described in #214, this is because aliases seem to be ignoring the runtime type.

**Comment by BrennanConroy** [comment]
Fixed in ad29a7b91a515ee96ff2efdee50f6005f3ac85ec
